packages/tools/src/utilities/getModalityUnit.ts

```diff
@@ -1,16 +1,30 @@
+import { metaData } from '@cornerstonejs/core';
+
 function getModalityUnit(
   modality: string,
   isPreScaled: boolean,
-  isSuvScaled: boolean
+  isSuvScaled: boolean,
+  imageId: string
 ): string {
   if (modality === 'CT') {
     return 'HU';
-  } else if (
-    modality === 'PT' &&
-    isPreScaled === true &&
-    isSuvScaled === true
-  ) {
-    return 'SUV';
+  } else if (modality === 'PT') {
+    if (isPreScaled === true) {
+      if (isSuvScaled === true) {
+        return 'SUV';
+      }
+
+      const petSeriesModule = metaData.get('petSeriesModule', imageId);
+      const units = petSeriesModule?.units;
+
+      if (units) {
+        return units;
+      } else {
+        return 'unitless';
+      }
+    } else {
+      return 'raw';
+    }
```

Comment:
can we please refactor this into handlePTModality? at the bottom of this function?

Reply:
I believe it can be written simpler like 

```js
function handlePTModality(isPreScaled, isSuvScaled, imageId, metaData) {
  if (!isPreScaled) {
    return 'raw';
  }
  
  if (isSuvScaled) {
    return 'SUV';
  }

  const petSeriesModule = metaData.get('petSeriesModule', imageId);
  return petSeriesModule?.units ?? 'unitless';
}
```